jsonschema-core.xml

```diff
@@ -1510,22 +1510,15 @@
                         <t>
                             Together with "$dynamicAnchor", "$dynamicRef" implements a cooperative
                             extension mechanism that is primarily useful with recursive schemas
-                            (schemas that reference themselves).  The extension point is defined with
-                            "$dynamicAnchor" and only exhibits runtime dynamic behavior when referenced
-                            with "$dynamicRef".
+                            (schemas that reference themselves).  The extension point is defined
+                            with "$dynamicAnchor", and only exhibits runtime dynamic behavior when
```

Comment:
The comma isn't necessary in the current form.  If you want to include a comma, the second half of the sentence needs to be a complete sentence.

```suggestion
                            with "$dynamicAnchor", and it only exhibits runtime dynamic behavior when
```